scss/scaffold.scss

```diff
@@ -71,10 +87,6 @@ tab {
 	grid-template-columns: 1fr max-content;
 }
 
-#tabpanel-metadata input {
```

Comment:
@dstillman

This is where the input height are changed: we used to explicitly set the height, and now that we are setting all form element e.g. button and input's height by px on Windows, I removed this style to prevent adding too much platform-specific styles.

Or if we want this back, we can treat Windows as an exception here.

Reply:
I think it was vaguely nicer before, but we could just add a little bit of padding below each row instead of changing the height of the textboxes themselves.